examples/ready-to-run/processing_example.py

```diff
@@ -22,43 +22,37 @@ def processing_of_monthly_data(simulation: pp.Simulation):
     # make label fit inside your plot
```

Comment:
We have discussed this one.
Maybe think of a way to simplify this.
Otherwise, this is fine